test/acceptance/docker-compose.yaml

```diff
@@ -9,4 +9,9 @@ services:
       CHROME_ARGS: "--headless --ignore-certificate-errors"
       BROWSERSTACK_USERNAME: ${BROWSERSTACK_USERNAME}
       BROWSERSTACK_ACCESS_KEY: ${BROWSERSTACK_ACCESS_KEY}
-      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
\ No newline at end of file
+      BROWSERSTACK_PARALLEL_RUNS: ${BROWSERSTACK_PARALLEL_RUNS}
+      JWT_SECRET: ${JWT_SECRET}
+      JWT_ISSUER: ${JWT_ISSUER}
+      NODE_TLS_REJECT_UNAUTHORIZED: "0"
```

Comment:
dont want to commit `NODE_TLS_REJECT_UNAUTHORIZED` - just keep this locally for the Kainos Zscaler issues